### PR TITLE
Added option, tuple2, and map including keys to TryUtil methods

### PR DESCRIPTION
**Pull request by kshakir**
Also removed out of date and empty changelog.

### CODE REVIEW

File: src/test/scala/lenthall/util/TryUtilSpec.scala [hunk 1]
```diff
@@ -38,6 +38,28 @@ class TryUtilSpec extends FlatSpec with Matchers {
     exception.throwables.head.getMessage should be("failed")
   }
 
+  it should "sequence successful nones" in {
+    val result = sequenceOption(None, "prefix")
```
Comment (mcovarr) [inline src/test/scala/lenthall/util/TryUtilSpec.scala]: it might be helpful for understanding to add ascriptions to these `result` variables

**Comment by mcovarr** [comment]
:+1:

**Comment by geoffjentry** [comment]
One of these days this thing will finally die the death it deserves but until then :+1:

**Comment by mcovarr** [comment]
🔥

**Pull request merged**